_posts/2020-01-08-#14053.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: pr
+date: 2020-01-08
+title: "Add address-based index (attempt 4?)"
+pr: 14053
+authors: [marcinja]
+components: ["utxo db and indexes"]
+host: jnewbery
+---
+
+_Review branch HEAD commit: ec5fbd0b90c98433979dd5a1d7f2b6254052cefa ([Tagged branch](https://github.com/bitcoin-core-review-club/bitcoin/tree/pr14053))_
```

Comment:
Just noticed this! I think ` ec5fbd0` would be enough here. Is the idea that participants should review this rather than the real PR, or use it for creating stable links in the meeting (and notes and questions)?

Reply:
Re-reading issue #90, you wrote "tag the PR branch before the review club, and use that tagged branch for all links". It might be good to clarify this so it's not confusing for the participants.

Reply:
The reason I'm adding this is that I expect there will be further changes on this PR before it gets merged. If anyone wants to read the review club notes in future, it'll be useful for them to be able to checkout the exact branch that was being discussed. Do you have any suggested wording so this isn't confusing for meeting attendees?

Reply:
Noting here for later reference:

"The PR commit HEAD was [commit hash](link to tag) at the time of this review club meeting".